### PR TITLE
Feature/111 clean up whole database

### PR DESCRIPTION
**Pull request by drekoumis**
Create API endpoint to clean up the whole database

### CODE REVIEW

**Review by fgypas — CHANGES_REQUESTED** [review]
I just tried the `api/clean-up-database` and I get the following:
`{"error":404,"msg":"Database is empty"}`, even though that the database is empty. Can you please have a look once more @drekoumis ?

**Comment by drekoumis** [comment]
Is any way to use table as input at function and use it as a variable in a query?

**Comment by drekoumis** [comment]
> 
> 
> I just tried the `api/clean-up-database` and I get the following:
> `{"error":404,"msg":"Database is empty"}`, even though that the database is empty. Can you please have a look once more @drekoumis ?

File: panoptes/routes/api.py [hunk 1]
```diff
@@ -76,8 +77,22 @@ def set_db_delete(workflow_id):
...
+
+
+@routes.route('/api/clean-up-database', methods=['GET'])
+def set_whole_db_delete():
+    if get_db_table_is_empty('Workflows'):
+        return jsonify({'error': 404, 'msg': 'Database is empty'})
+    else:
+        delete=delete_whole_db()
+        if delete:
+            return jsonify({'msg': 'Clean Up The Whole Database Complete Correctly'})
```
Comment (fgypas) [inline panoptes/routes/api.py:96]: Can you please change the message to:
"Database clean up is complete"

File: panoptes/routes/api.py [hunk 2]
```diff
@@ -76,8 +77,22 @@ def set_db_delete(workflow_id):
     elif(get_db_workflows_by_status(workflow_id)=='Running'):
         return jsonify({'error': 'Delete Rejected', 'msg': 'You cannot delete Running Workflow '})
     else:        
-        delete=del_db_wf(workflow_id)
+
```
Comment (fgypas) [inline panoptes/routes/api.py:80]: Remove the empty lines

**Review by fgypas — CHANGES_REQUESTED** [review]
@drekoumis Can you please make these minor changes?

**Review by fgypas — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**